plugin/templates/insert_builder.mustache

```diff
@@ -15,7 +15,7 @@ public final Builder set{{fieldNameUpper}}With{{resourceTypeClassName}}({{resour
 /**
  * @deprecated Use the following instead:
  * <pre><code>
- * {{resourceTypeClassName}}.parse(protoObject.get{{fieldNameUpper}}());
+ * {{resourceTypeClassName}}.parse(protoBuilder.get{{fieldNameUpper}}());
```

Comment:
Does it make sense for this to be protoBuilder, when we are parsing from it, not setting anything?

This is pretty minor, so if this is difficult to fix then I wouldn't worry about it.